<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBookDialog</class>
 <widget class="QDialog" name="PhoneBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelNama">
     <property name="text">
      <string>Nama</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditNama"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTelp">
     <property name="text">
      <string>Telp</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditTelp"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEmail">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditEmail"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>PhoneBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
